From 11018606b9fcf5f56deb141e16d5b752651a1459 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Fri, 15 May 2020 17:46:02 +0000
Subject: [PATCH] program_family.cpp: include string and stdexcept to fix build
 with gcc-10

* fixes:
  anbox/3.0+gitAUTOINC+0a49ae08f7-r0/git/src/anbox/graphics/program_family.cpp:39:20: error: 'runtime_error' is not a member of 'std'
  anbox/3.0+gitAUTOINC+0a49ae08f7-r0/git/src/anbox/graphics/program_family.cpp:39:39: error: 'string' is not a member of 'std'
  anbox/3.0+gitAUTOINC+0a49ae08f7-r0/git/src/anbox/graphics/program_family.cpp:86:18: error: 'runtime_error' is not a member of 'std'
  anbox/3.0+gitAUTOINC+0a49ae08f7-r0/git/src/anbox/graphics/program_family.cpp:86:37: error: 'string' is not a member of 'std'

Upstream-Status: Submitted [https://github.com/anbox/anbox/pull/1456]

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/anbox/graphics/program_family.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/anbox/graphics/program_family.cpp b/src/anbox/graphics/program_family.cpp
index c0bf8a0..4d85f47 100644
--- a/src/anbox/graphics/program_family.cpp
+++ b/src/anbox/graphics/program_family.cpp
@@ -20,6 +20,9 @@
 
 #include "anbox/graphics/emugl/DispatchTables.h"
 
+#include <string>
+#include <stdexcept>
+
 namespace anbox {
 namespace graphics {
 void ProgramFamily::Shader::init(GLenum type, const GLchar* src) {
